x/dkim/types/codec.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+	// this line is used by starport scaffolding # 1
+)
+
+var (
+	amino    = codec.NewLegacyAmino()
+	AminoCdc = codec.NewAminoCodec(amino)
+)
+
+func init() {
```

Comment:
This should only be needed if we need special signing codecs. For this module, I don't believe that is true.